src/reporter.ts

```diff
@@ -120,11 +127,19 @@ class RPReporter implements Reporter {
     this.launchId = tempId;
   }
 
+  isMapElementExist(suite: Map<string, Suite>, title: string): Suite {
```

Comment:
This function not only check the item existense, but also try to find it, so please rename it to smth like `findTestItem`.
Also its strange that you call parameter `suite` as you can use here other testItems.